__tests__/user/user.mutation.test.js

```diff
@@ -1,28 +1,33 @@
 /* eslint-disable no-undef */
-const { gql } = require('@apollo/client');
-const client = require('../../utils/apollo-test-client');
 const { adminUser, superAdminUser, newAdmin } = require('./user.variables');
-const adminLogin = require('../helpers/admin-login');
+const { adminLogin, setupApp } = require('../helper-functions');
 const {
   INPUT_NOT_VALID,
   INVALID_ADMIN_INVITATIONAL_TOKEN,
   USER_ALREADY_EXIST,
 } = require('../../error-messages/user.messages');
 
+const { gql } = require('@apollo/client');
+const client = require('../../utils/apollo-test-client');
```

Comment:
Why do we need lines `#10, #11` if we are not using client anymore?

Reply:
We use gql variable in requests to the server.